be/src/olap/reader.cpp

```diff
@@ -599,13 +599,12 @@ OLAPStatus Reader::_init_params(const ReaderParams& read_params) {
 
     if (_tablet->tablet_schema().has_sequence_col()) {
         _sequence_col_idx = _tablet->tablet_schema().sequence_col_idx();
-        if (_sequence_col_idx != -1) {
-            for (auto col : _return_columns) {
-                // query has sequence col
-                if (col == _sequence_col_idx) {
-                    _has_sequence_col = true;
-                    break;
-                }
+        DCHECK_NE(_sequence_col_idx, -1);
```

Comment:
Why using DCHECK here?

Reply:
Because `has_sequence_col ` in line 600 is `inline bool has_sequence_col() const { return  _sequence_col_idx != -1; }`, so `_sequence_col_idx` shoule always be `-1`.